### PR TITLE
Support neterror.xhtml functionality. JB#54831 OMP#JOLLA-223

### PR DESCRIPTION
**Pull request by llewelld**
Adds hooks to support the functionality on the new neterror.xhtml page (shown when the browser can't open a page for whatever reason).

The new functionality:
1. The "Search" box is pre-filled with what the user entered in the address bar.
2. The "Try again" button refreshes the page.
3. The "Wifi" button opens up the connection selector.

These changes build on the changes in #7, so only the most recent commit is relevant here.

To test: try opening a page from a domain that doesn't exist (e.g. abd.def). Try opening a page when there's no network connection.

### CODE REVIEW

**Comment by llewelld** [comment]
This PR relies on the changes in https://github.com/sailfishos/sailfish-browser/pull/896

**Comment by llewelld** [comment]
Updated to fix errors identified by Chris.

**Comment by llewelld** [comment]
Updated to prevent the online status being confused by Developer Mode.

File: overrides/aboutCertError.xhtml [hunk 1]
```diff
@@ -64,73 +62,91 @@
...
           if (node.textContent == "#1")
             node.textContent = location.host;
           else
-            for(var i = 0; i < node.childNodes.length; i++)
+            for (var i = 0; i < node.childNodes.length; i++)
               replaceWithHost(node.childNodes[i]);
-        };
-        replaceWithHost(intro);
+        }
+  replaceWithHost(intro);
```
Comment (chriadam) [inline overrides/aboutCertError.xhtml:78]: whitespace / alignment

**Review by chriadam — APPROVED** [review]
LGTM, works nicely.

**Review by okodron — APPROVED** [review]

File: overrides/aboutCertError.xhtml (hunk 1 above)
Comment (llewelld) [reply overrides/aboutCertError.xhtml]: Thank you! Fixed.

**Comment by llewelld** [comment]
Rebased.

**Pull request merged**